.github/workflows/platform.publish-module-index-json.yml

```
@@ -4,6 +4,12 @@ on:
   schedule:
     - cron: 45 11 * * * # Run daily at 3:45 AM PST
   workflow_dispatch:
+    inputs:
+      doNotMergeWithLastModuleIndexJsonFileVersion:
+        type: boolean
+        description: "doNotMergeWithLastModuleIndexJsonFileVersion | If toggled, the last version of the moduleIndex.json file that is downloaded from the storage account will not be merged with the current generated moduleIndex.json file. Effectively creating the moduleIndex.json from scrath based on what is in BRM at the time of the run."
```

Just tossing this around if it makes sense 
```suggestion
      regenIndexFromBRM:
        type: boolean
        description: "regenIndexFromBRM | If toggled, regenerates the moduleIndex.json from scratch based on what is in BRM at the time of the run, instead of integrating it with the last version of the moduleIndex.json file that is downloaded from the storage account."
```

---

Anyway I guess we can even be less detailed given this is only intended to be run by few people. I'm not sure if the following is correct, I'm just guessing from what I read
```suggestion
      regenIndexFromBRM:
        type: boolean
        description: "regenIndexFromBRM | Regenerate the moduleIndex.json file from scratch based on what is in BRM at the time of the run, instead of integrating it with its previous version."
```

---

took your second suggestion :)